tox.ini

```diff
@@ -53,7 +51,7 @@ commands =
     # uncomment the following line if this charm owns a lib
     # codespell {[vars]lib_path}
     codespell {toxinidir}/. --skip {toxinidir}/./.git --skip {toxinidir}/./.tox \
-      --skip {toxinidir}/./build --skip {toxinidir}/./lib --skip {toxinidir}/./venv \
+      --skip {toxinidir}/./build --skip {toxinidir}/./lib --skip {toxinidir}/./venv* \
```

Comment:
does this affect anything?  shouldn't the original skip the venv dir already?

Reply:
yes, I have venv(s) with the name of python version in it